lib/proto.js

```diff
@@ -91,13 +91,26 @@ function child (bindings) {
       instance[serializersSym][bks] = bindings.serializers[bks]
     }
   } else instance[serializersSym] = serializers
+  if (bindings.hasOwnProperty('formatters')) {
+    instance[formattersSym] = {
+      level: bindings.formatters.level || formatters.level,
+      bindings: bindings.formatters.bindings || resetChildingsFormatter,
+      log: bindings.formatters.log || formatters.log
+    }
+  } else {
+    instance[formattersSym] = {
+      level: formatters.level,
+      bindings: resetChildingsFormatter,
+      log: formatters.log
+    }
```

Comment:
Instead of creating two objects at two separate lines, could you wrap them in a single factory? It would help in them having the same shape.